style.css

```diff
@@ -672,3 +672,82 @@ header nav div .menu-bar {
   background: #e5e5e5;
   color: #5e6c84;
 }
+
+.contact-form {
+  width: 375px;
+  height: 779px;
+  padding-top: 103px;
+  background: #6070ff;
+  background-image: url(image/contact\ form\ shapes-mobile@2x.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  text-align: center;
+  align-items: center;
+  color: #fff;
+  border-top-left-radius: 70px;
+  display: inline-block;
+}
```

Comment:
- [ ] While you did a wonderful job styling your project :100:  you should have a `white` background instead of `gray` on the contact form section.

**Figma design**

![Screenshot from 2023-01-17 20-08-31](https://user-images.githubusercontent.com/105117832/213056014-cd1f59f4-73af-49f7-86ad-1ef3ee858232.png)

**Your design**

![Screenshot from 2023-01-17 20-09-11](https://user-images.githubusercontent.com/105117832/213056089-6efba059-77e2-4ece-acdc-b648b4b5888f.png)